enterprise/server/githubapp/githubapp.go

```diff
@@ -355,33 +476,13 @@ func (a *GitHubApp) GetRepositoryInstallationToken(ctx context.Context, repo *ta
 	return tok.GetToken(), nil
 }
 
-func (a *GitHubApp) GetGitHubAppInstallations(ctx context.Context, req *ghpb.GetAppInstallationsRequest) (*ghpb.GetAppInstallationsResponse, error) {
-	u, err := a.env.GetAuthenticator().AuthenticatedUser(ctx)
-	if err != nil {
-		return nil, err
-	}
-	// List installations linked to the org.
-	rq := a.env.GetDBHandle().NewQuery(ctx, "githubapp_get_installations").Raw(`
-		SELECT *
-		FROM "GitHubAppInstallations"
-		WHERE group_id = ?
-		ORDER BY owner ASC
-	`, u.GetGroupID())
-	res := &ghpb.GetAppInstallationsResponse{}
-	err = db.ScanEach(rq, func(ctx context.Context, row *tables.GitHubAppInstallation) error {
-		res.Installations = append(res.Installations, &ghpb.AppInstallation{
-			GroupId:        row.GroupID,
-			InstallationId: row.InstallationID,
-			Owner:          row.Owner,
-		})
-		return nil
-	})
-	if err != nil {
-		return nil, status.InternalErrorf("failed to get installations: %s", err)
-	}
-	return res, nil
-}
-
+// LinkGitHubAppInstallation imports an installed GitHub app to BuildBuddy.
+//
+// After a user has installed the BuildBuddy Github app from the GitHub side,
+// this imports the data to BuildBuddy and saves it to the database.
+// This will fail if the user didn't install the app in GitHub.
+//
+// Note that GitHub is always the ultimate source of truth.
```

Comment:
may be worth clarifying this a bit, since the DB row is the source of truth for the installation-group link, but not whether the installation actually exists.

```suggestion
// Note that GitHub is always the source of truth for whether the app is installed.
// Linking an installation associates an installation with a group, but a linked installation
// existing doesn't imply that the installation is still valid.
```